### ISSUE TITLE
ARIA compliance

### ISSUE THREAD

**Issue by GoogleCodeExporter** [mar10/dynatree#383]
```
The tree is not natively ARIA compliant and as such it does not comply with 
Section 508 of the US Rehabilitation Act. Consequently, it can not be cleared 
for use on any US federal website. The main practical issue is that the tree 
does not work properly with common screen readers such as JAWS and ChromeVox. I 
suspect some small modifications to the code would render it compliant. See 
http://test.cita.uiuc.edu/aria/tree/index.php for an example.


```

Original issue reported on code.google.com by `john.cla...@cornerstonenw.com` on 26 Nov 2012 at 2:58
